app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :"views/tweets/tweets"
+     else
+       redirect to "/login"
+     end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      @user = current_user
+      erb :'views/tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/new/tweet' do
+    if !params[:tweet][:content].blank?
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.